api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -346,7 +346,10 @@ private void validateWorkspaceTierAccess(DbWorkspace dbWorkspace) {
     List<String> accessTiers = accessTierService.getAccessTierShortNamesForUser(userProvider.get());
 
     if (!accessTiers.contains(workspaceAccessTier)) {
-      throw new ForbiddenException("User does not have access to the workspace tier");
+      throw new ForbiddenException(
+          String.format(
+              "User with username %s does not have access to the workspace '%s' tier",
```

Comment:
How about:
`User with username joel does not have access to the 'controlled' access tier required by workspace 'CT Test'`

Reply:
just the first name for user?

Reply:
full email for username is probably better 👍

Reply:
<img width="1035" alt="Screen Shot 2021-12-09 at 12 46 41 PM" src="https://user-images.githubusercontent.com/34481816/145448718-53b6f0cd-86b8-4d68-802c-580704daee2d.png">